src/xma/src/xmaapi/xma_utils.cpp

```diff
@@ -28,11 +29,79 @@
 #include <boost/filesystem/path.hpp>
 #include <iostream>
 #include <sstream>
+#include <bitset>
 
 #define XMAUTILS_MOD "xmautils"
 
 extern XmaSingleton *g_xma_singleton;
 
+namespace xma_core {
+    std::map<XmaSessionType, const char*> sessionMap = {
```

Comment:
Should be declared `static const`.  

in addition, it would be best to change the "char string" value to actual contain a string, for example:

`static const std::map<XmaSessionType, std::string> sessionMap = {`

Reply:
done